test.js

```diff
@@ -459,26 +460,20 @@ test('test ignoring pairs with the wrong type', function (t, done) {
   function write (db) {
     var ws = db.createWriteStream()
     ws.on('error', function (err) {
-      t.notOk(err, 'no error')
+      t.equal(err.message, '`type` must be \'put\' or \'del\'', 'should error')
```

Comment:
Should also assert that an error event is indeed emitted, because there is no `t.plan`.

Reply:
Gah, wish I could do `t.plan()` instead of using `t.end()` here.